config/config_test.go

```diff
@@ -363,37 +364,70 @@ func TestValidateReferencesExist(t *testing.T) {
 	}
 }
 
-func TestUpdate_validateNames(t *testing.T) {
+func TestValidateName(t *testing.T) {
+	stringOfLength := func(length int) string {
```

Comment:
IMO this function doesn't need to exist as it is only called once right now.

In the future when someone is adding a new test to this list and/or debugging a failure that isn't the "Names can't be too long" test, this adds unnecessary complexity for that person, reducing their performance.